src/externs/js-cookie.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @fileoverview js-cookie library extern.
+ */
+
+/**
+ * @constructor
+ * @return {!Cookies}
+ */
+function Cookies() {}
+
+/**
+ * @param {string} name
+ * @param {string=} value
+ * @param {Object=} opt_params
+ */
+Cookies.get = function(name, value, opt_params) {};
+
+/**
+ * @param {string} name
+ * @param {string=} value
```

Comment:
Probably you want the second arg of  .set() to be non optional.

On Sunday, March 20, 2016, Liana Yang <notifications@github.com> wrote:

> In src/externs/js-cookie.js
> <https://github.com/ViDA-NYU/genotet/pull/145#discussion_r56759415>:
>
> > +/**
> > + * @constructor
> > + * @return {!Cookies}
> > + */
> > +function Cookies() {}
> > +
> > +/**
> > + * @param {string} name
> > + * @param {string=} value
> > + * @param {Object=} opt_params
> > + */
> > +Cookies.get = function(name, value, opt_params) {};
> > +
> > +/**
> > + * @param {string} name
> > + * @param {string=} value
>
> It will return the value of 'hello'.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/ViDA-NYU/genotet/pull/145/files/83dd548aa3090df14c14b115859530f16cb2a2a5#r56759415>
>